src/components/FilterButton.vue

```diff
@@ -1,31 +1,59 @@
 <template>
   <button
     class="filter-button"
+    :class="{ hover }"
     @click="$emit('click', $event)"
+    @mouseover="hover = true"
+    @mouseleave="hover = false"
   >
     <slot />
   </button>
 </template>
 
+<script>
+export default {
+  data() { return { hover: false }; },
+};
+</script>
+
 <style lang="scss" scoped>
 .filter-button {
   border-radius: 1rem;
   font-size: 0.8rem;
   font-weight: 700;
-  color: $light_font_color;
+  color: $tip_note_color;
   display: inline-block;
   padding: 0.35rem 0.7rem;
   border: none;
   background-color: transparent;
   transition: color 0.3s ease-in-out, background-color 0.3s ease-in-out;
 
-  &:hover {
-    color: $tip_note_color;
+  svg {
+    color: $pure_white;
+    opacity: 0.7;
+    transition: opacity 0.3s ease-in-out;
+    height: 24px;
+    width: auto;
+  }
+
+  &.hover {
```

Comment:
```suggestion
  &:hover {
```
won't it work the same, but without the extra state?